website/docs/guides/data-interactions/queries.mdx

```diff
@@ -382,6 +382,25 @@ Similarly, you can filter the documents based on a range of values. For example,
 }
 ```
 
+Note: the query above can also be written in a less verbose way as shown below. Under the hood, this filtering condition 
+will use the same `and` logic as the example written above:
+
+```json
+{
+  "input":{      
+    "where":{
+      "created_at":{
+        "greaterThenOrEqualTo":"2023-04-01",
+        "lessThanOrEqualTo":"2023-08-01"
+      }
+    }    
+  }
+}
+```
+Check out the [reference](../../api/runtime/schema.mdx#logical-conditions) for more examples of how you can write queries with logical conditions.
+
+
```

Comment:
You can also add either in this example or as a separate one that you can have multiple fields in a single "where".
I'm using this example in the tutorial:
```
{
   "input":{
      "where":{
         "created":{
            "greaterThanOrEqualTo":"2023-01-01",
            "lessThan":"2023-12-31"
        },
      	"tag":{
        	"in": ["Decentralization", "announcement"]
        }
      }
  	}
}
```

Or if you want to keep things simple, then all good in this piece of code.